Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -297,6 +297,8 @@ var ScrollView = React.createClass({
     /**
      * A RefreshControl component, used to provide pull-to-refresh
      * functionality for the ScrollView.
+     *
+     * See [RefreshControl](/react-native/docs/refreshcontrol.html).
```

Comment:
I was wondering if it would be confusing to people inside facebook reading the code. if you're ok with it let's ship this.